### ISSUE TITLE
Lock and unlock work only with the first argument

### ISSUE THREAD

**Issue by sandrineBeauche** [ow2-proactive/scheduling#2294]
The lock and unlock commands are executed correctly only for the first given node in the arguments in the interaction client. But the client allows to give several nodes at the same time. 

The return of the command shows that it is performed successfully, but  when I verify on the web resource manager, only the first node in the argument list has been locked or unlocked.

**Comment by lpellegr** [comment]
@sandrineBeauche Could you please add a copy/paste of the command used and its output. It will be helpful when we will come back on the issue.

**Comment by sandrineBeauche** [comment]
For example, in the interactive client:

```
> locknodes('pnp://pc-sandrinoun:7313/local-LocalNodes-7', 'pnp://pc-sandrinoun:7355/local-LocalNodes-5')
Node(s) locked successfully.
```

Only the node pnp://pc-sandrinoun:7313/local-LocalNodes-7 is locked.

**Comment by sandrineBeauche** [comment]
This problems occurs only in the interactive mode. In the non-interactive mode, all the specified nodes are locked and unlocked.

**Comment by sandrineBeauche** [comment]
Bug fixed and merged into master with this commit: 2408e59004258408f01ae5291a7fac093e50bb09
